views/js/qtiCreator/context/qtiCreatorContext.js

```diff
@@ -96,6 +96,7 @@ define([
              * @returns {Promise}
              */
             destroy: function destroy() {
+                var self = this;
```

Comment:
why not use arrow func and then we dont need to store scope anymore

Reply:
Yes, I could have updated the whole file to be ES6. But this is only a fix, and since it took me some time to figure out the issue, I did not want to break something else aside by changing too much things.